THIRD_PARTY_LICENSES.txt

```diff
@@ -5434,7 +5434,7 @@ Copyright 2020 Google LLC.
 Copyright 2020 Google LLC. All Rights Reserved.
 Copyright 2020 Google LLC. All Rights Reserved.\n" +
 Copyright 2020 Gregor Martynus
-Copyright 2020 Intel Coporation.
+Copyright 2020 Intel Corporation.
```

Comment:
I'm confident that this name was wrong. Although I'm not sure where it's originally from. https://www.intel.com/content/www/us/en/legal/trademarks.html

Reply:
This appears to come from kubernetes, I've submitted a PR to fix it there.

Reply:
And it was merged